android/src/main/java/dev/openfeature/sdk/async/Extensions.kt

```diff
@@ -48,9 +48,9 @@ internal fun FeatureProvider.observeProviderReady() = observe<OpenFeatureEvents.
 Observe events from currently configured Provider.
  */
 @OptIn(ExperimentalCoroutinesApi::class)
-fun OpenFeatureAPI.observeEvents(): Flow<OpenFeatureEvents> {
+internal inline fun <reified T : OpenFeatureEvents> OpenFeatureAPI.addHandler(): Flow<T> {
```

Comment:
IMO the name of this function (combined with what it does) does not spark joy.
I'll give it some more thought but my immediate reaction is that we should embrace the platform and the conventions instead. 
Of course this should then be highlighted in the readme/documentation.

Reply:
I am also conflicted. Would you then prefer to leave `observe()`? I'd be happy with that

Reply:
Maybe we can have both so we don't need to decide 😸?

Reply:
Pulling in @vahidlazio for an extra opinion

Reply:
Agreed! And PR updated